From b90109511ed7a067c18a7261430119890aa0c1d3 Mon Sep 17 00:00:00 2001
From: Miles Hu <milehu@codeaurora.org>
Date: Sat, 22 Jun 2019 13:45:11 -0700
Subject: [PATCH 2/2] ath11k: skip regular data packet to monitor interface in
 co-exist mode.

Add SKIP flag to every rx regular data packet

Signed-off-by: Miles Hu <milehu@codeaurora.org>
---
 drivers/net/wireless/ath/ath11k/dp_rx.c | 1 +
 1 file changed, 1 insertion(+)

--- a/drivers/net/wireless/ath/ath11k/dp_rx.c
+++ b/drivers/net/wireless/ath/ath11k/dp_rx.c
@@ -2164,6 +2164,7 @@ static void ath11k_dp_rx_pre_deliver_ams
 		} else {
 			rxs->flag |= RX_FLAG_ALLOW_SAME_PN;
 		}
+		rxs->flag |= RX_FLAG_SKIP_MONITOR;
 
 		status = IEEE80211_SKB_RXCB(msdu);
 		*status = *rxs;
